Gruntfile.js

```diff
@@ -132,13 +132,7 @@ module.exports = function (grunt) {
                     ['babelify'],
                     [(file) => {
```

Comment:
have you tried dropping this whole thing? looks like a possible noop

Reply:
Ah yeah, will try checking.